frontend/scripts/buildCache.ts

```diff
@@ -39,21 +52,39 @@ async function testAPIAccess() {
 function cacheRPCensus() {
   return seq(...DATASETS.map(_ => async () => {
     console.log('  Getting RPs and Census for ' + safeDatasetHint(_))
-    await getRepresentativePoints({ service_area_ids: _.serviceAreaIds })
-    await getCensusData({ service_area_ids: _.serviceAreaIds })
+    let representativePoints = await getRepresentativePoints({ service_area_ids: _.serviceAreaIds })
+    let census = await getCensusData({ service_area_ids: _.serviceAreaIds })
+    if (uploadToS3) {
+      let rpParams = { Bucket: s3Bucket, Key: getS3Key(getStaticRPUrl(_)), Body: JSON.stringify(representativePoints), ContentType: 'application/json', ACL: 'public-read' }
+      let censusParams = { Bucket: s3Bucket, Key: getS3Key(getStaticDemographicsUrl(_)), Body: JSON.stringify(census), ContentType: 'application/json', ACL: 'public-read' }
+      s3.putObject(rpParams, s3Callback)
+      s3.putObject(censusParams, s3Callback)
+    }
   }))
 }
 
 function cacheAdequacies() {
   return seq(...DATASETS.map(dataset => async () =>
     seq(...METHODS.map(method => async () => {
       console.log('  Getting Adequacy for ' + safeDatasetHint(dataset) + ' for ' + method)
-      return getAdequacies({
```

Comment:
don't you still need a return in case you're not uploading to s3?

Reply:
what?

Reply:
where's the else if not upload to s3?

Reply:
The getAdequacies happens in both cases...